### ISSUE TITLE
Make Dataverse Previewers support more media types

### ISSUE THREAD

**Issue by philippconzett** [GlobalDataverseCommunityConsortium/dataverse-previewers#64]
As a potential dataset downloader, I'd like to be able to preview more media types, in particular the following ones:

image/tiff
audio/flac
text/markdown
text/semicolon-separated-values

Note on text/semicolon-separated-values:
When testing in the Harvard Demo Dataverse, semicolon-separated .csv files are not ingested and previewed. Semicolon-separated .txt files are not ingested, but previewed as plain text files. See this test dataset:
[https://demo.dataverse.org/dataset.xhtml?persistentId=doi:10.70122/FK2/H0U0FQ](https://demo.dataverse.org/dataset.xhtml?persistentId=doi:10.70122/FK2/H0U0FQ)

**Comment by adam3smith** [comment]
Do you want the markdown rendered or just displayed as text? The latter
should be trivial

**Comment by philippconzett** [comment]
Yes, that would be good. If it at a later stage will be possible to be displayed as Markdown, that would be even better :)

**Comment by kaitlinnewson** [comment]
For Markdown, perhaps something like [marked](https://github.com/markedjs/marked) could be used for rendering?

**Comment by kaitlinnewson** [comment]
I did a quick test, and Markdown files should work as-is in the text previewer (but are not rendered), e.g.:

https://dataverse.scholarsportal.info/preview/previewers/TextPreview.html?fileid=110999&siteUrl=https://dataverse.scholarsportal.info&datasetid=110661&datasetversion=3.0&locale=en

**Comment by pdurbin** [comment]
It would be cool so have some sort of toggle to switch between raw text and rendered HTML but this is a great start!
